src/components/_global/Modal/ModalSwitch/index.js

```diff
@@ -5,8 +5,10 @@ import DeletePoints from 'views/Trace/DeletePoints';
 import RecordsTable from 'components/_shared/RecordsTable';
 import StageForPublishing from 'views/Trace/StageForPublishing';
 import PublishData from 'views/Publish/PublishData';
+import ForgotPassword from 'views/Authentication/ForgotPassword';
 
 const ModalSwitch = ({ status, mode }) => {
+  console.log(status);
```

Comment:
console log police you are under arrest! 👮‍♂️

Reply:
Guilt. Send me to the big house.